### ISSUE TITLE
Pretrained model normalization values link is broken

### ISSUE THREAD

**Issue by abhishri-medewar** [pytorch/pytorch.github.io#1331]
## 📚 Documentation

The documentation for the pretrained models states that they were trained with hard-coded normalization values, as described in this [Link](https://pytorch.org/docs/master/torchvision/models.html). However, when I click on the link, it says "Oops, you've reached a dead end".

Could someone please update the link?

**Comment by svekars** [comment]
Thanks for submitting this issue. Can you please post the link to the page on which the broken link is located? Thanks!

**Comment by abhishri-medewar** [comment]
https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html
